osbs/build/user_params.py

```diff
@@ -129,8 +129,9 @@ def __init__(self, build_json_store=None):
             "arrangement_version",
             allow_none=True,
             default=REACTOR_CONFIG_ARRANGEMENT_VERSION)
+        self.build_from = BuildParam('build_from')
         self.build_image = BuildParam('build_image')
-        self.build_imagestream = BuildParam('build_imagestream')
+        self.build_is_imagestream = BuildParam('build_is_imagestream', default=False)
```

Comment:
I'd name it either build_image_is_imagestream or buildroot_is_imagestream, or just image_is_imagestream
I like the most buildroot_is_imagestream :)

Reply:
Done.